enterprise/server/ociregistry/ociregistry_test.go

```diff
@@ -0,0 +1,165 @@
+package ociregistry_test
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net"
+	"net/http"
+	"runtime"
+	"sync"
+	"testing"
+
+	"github.com/buildbuddy-io/buildbuddy/enterprise/server/ociregistry"
+	"github.com/buildbuddy-io/buildbuddy/enterprise/server/util/oci"
+	"github.com/buildbuddy-io/buildbuddy/server/environment"
+	"github.com/buildbuddy-io/buildbuddy/server/testutil/testenv"
+	"github.com/buildbuddy-io/buildbuddy/server/testutil/testport"
+	"github.com/buildbuddy-io/buildbuddy/server/testutil/testregistry"
+	"github.com/buildbuddy-io/buildbuddy/server/util/testing/flags"
+	v1 "github.com/google/go-containerregistry/pkg/v1"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	rgpb "github.com/buildbuddy-io/buildbuddy/proto/registry"
+)
+
+type requestCounter struct {
```

Comment:
nit: I think you can use an atomic.Int32 here if you want. See https://github.com/buildbuddy-io/buildbuddy/blob/46a78118effd410cf8146026e0bbf845150fe3b0/enterprise/server/byte_stream_server_proxy/byte_stream_server_proxy_test.go#L96

Reply:
Fixed! Thanks for the pointer.